tests/pebble.rs

```diff
@@ -185,6 +150,82 @@ impl PebbleEnvironment {
         Ok(())
     }
 
+    /// Create and finalize an ACME order for the given `Account` and `Identifier`s using
+    /// the specified `ChallengeType`.
+    ///
+    /// The `provision_fn` will be invoked to set up the appropriate challenge response
+    /// based on the `Challenge`/`KeyAuthorization`.
+    ///
+    /// Returns the issued certificate chain unless an error occurs.
+    async fn complete_order<'a>(
+        &'a self,
+        account: &mut Account,
+        identifiers: &[String],
+        chal_type: ChallengeType,
+        provision_fn: impl Fn(
+            String,
+            &Challenge,
+            KeyAuthorization,
+        ) -> Pin<
+            Box<dyn Future<Output = Result<(), Box<dyn StdError + 'static>>> + Send + 'a>,
```

Comment:
What's the point of boxing the return future?

Reply:
This fell away.